RELEASENOTES.md

```diff
@@ -1,3 +1,3 @@
 - Added logic to check if a target repo exists before generating GHES archives
-- Added logic to ensure target org exists before generating GHES archives
-
+- Fixed a bug when reclaiming a single mannequin with --mannequin-user and --target-user parameters. They were not being passed to the reclaim command
```

Comment:
I suggest mentioning the command, enclosing the parameters in backticks and dropping the explanation of why it was broken (which only really matters for us!) :heart:

```suggestion
- Fixed reclaiming a single mannequin using `reclaim-mannequin` with the `--mannequin-user` and `--target-user` parameters
```

Reply:
@timrogers done